packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -616,6 +616,53 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 		this[__private__dont__use].FS.rename(fromPath, toPath);
 	}
 
+	/** @inheritDoc */
+	@rethrowFileSystemError('Could not copy "{path}"')
+	cp(fromPath: string, toPath: string, recursive = false) {
+		const FS = this[__private__dont__use].FS;
+
+		const fromStat = FS.stat(fromPath);
+
+		// Emscripten directories will have the sticky-bit set
+		// https://linux.die.net/man/1/chmod
+		const fromIsDir = fromStat.mode & 0o40000;
+
+		let toExists: boolean;
+
+		try {
+			FS.stat(toPath);
+			toExists = true;
+		} catch {
+			toExists = false;
+		}
+
+		if (!fromIsDir) {
+			const file = FS.readFile(fromPath, { encoding: 'binary' });
```

Comment:
Isn't `{ encoding: 'binary' }` the default? Also, let's use `this` functions whenever possible, e.g. there's `this.readFileAsBuffer()`

Reply:
https://github.com/WordPress/wordpress-playground/blob/cc576fc972ceaabeba7660aaa39494c95aca2de8/packages/php-wasm/universal/src/lib/base-php.ts#L656

Reply:
Thanks! What about other places where `this` functions could be applicable over custom-built logic?